geode-web/src/test/java/org/apache/geode/management/internal/cli/commands/RebalanceCommandOverHttpDistributedTest.java

```diff
@@ -19,14 +19,13 @@
 import org.junit.experimental.categories.Category;
 
 import org.apache.geode.test.junit.categories.DistributedTest;
-import org.apache.geode.test.junit.categories.FlakyTest;
 import org.apache.geode.test.junit.categories.GfshTest;
 
-@Category({DistributedTest.class, FlakyTest.class, GfshTest.class}) // GEODE-3530
+@Category({DistributedTest.class, GfshTest.class})
 @SuppressWarnings("serial")
 public class RebalanceCommandOverHttpDistributedTest extends RebalanceCommandDistributedTest {
 
   @ClassRule
   public static ProvideSystemProperty provideSystemProperty =
-      new ProvideSystemProperty(CliCommandTestBase.USE_HTTP_SYSTEM_PROPERTY, "true");
+      new ProvideSystemProperty("useHTTP", "true");
```

Comment:
this system property would have effect at once you refactored RebalanceCommandDistributedTest to not to use CliCommandTestBase, we will have to use parameters in RebalanceCommandDistributedTest to do this. See ChangeLogLevelCommandDUnitTest for example